### PR TITLE
DMP for plant sciences

### PR DESCRIPTION
**Pull request by erlefloch**
- Add a DMP section in the Plant Sciences domain page
- Shorten the DMP section of plant pheno assembly and make it link to the plant science page
- Add a DMP section to plant geno assembly and make it link to the plant science page

### CODE REVIEW

File: pages/tool_assembly/plant_genomics_assembly.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ All the components of this tool assembly are publicly available, but most requir
 ## For what purpose can you use the plant genomics tool assembly?
 {% include image.html file="plant_genomics.svg" caption="Figure 1. The plant genomics tool assembly." alt="Tools and resources used in managing plant genomics and genotyping data." %}
 
+### Data management planning
```
Comment (korbinib) [inline pages/tool_assembly/plant_genomics_assembly.md:28]: Should the tool (DSW?) also be mentioned in the text here?

File: pages/tool_assembly/plant_phenomics_assembly.md [hunk 2]
```diff
@@ -40,11 +40,7 @@ All the components of the plant phenomics tool assembly are publicly available a
 
 ### Data management planning
 
-The general principles to be considered are described in the [Plant Science domain page](plant_sciences) and in particular in its section dedicated to [plant phenotyping data](plant_sciences#phenotyping-metadata-collection-and-publication). In a nutshell:
-* the phenotyping data must be described following the [MIAPPE](https://www.miappe.org/) data standard;
-* special attention should be given to the identification and description of the [biological material](plant_sciences#plant-biological-materials-metadata-collection-and-sharing) and the [observation variables](plant_sciences#phenotyping-metadata-collection-and-publication).
-
-The general principles for data management planning and available tools are described in the RDMkit [data management plan page](data_management_plan). The knowledge model of the data management planning application [Data Stewardship Wizard (DSW)](https://ds-wizard.org/) was reviewed for compliance with the needs of the Plant Sciences community.
+The general principles to be considered are described in the [Plant Science domain page](plant_sciences).
```
Comment (korbinib) [inline pages/tool_assembly/plant_phenomics_assembly.md:43]: Should the tool (DSW?) also be mentioned in the text here?

**Review by korbinib — COMMENTED** [review]
should the tool(s) also be mentioned in the assembly pages and tables?

**Comment by bedroesb** [comment]
Does this mean that there are also tools to be added/updated in the tools table? I am thinking to maybe wait with this pull request until we have the new way of tool tagging merged :thinking:

**Comment by erlefloch** [comment]
@bedroesb  yes, sure! We can wait for the new way of tagging tools.

**Comment by bedroesb** [comment]
@erlefloch Th long awaited new way of tagging tools is live! The new contribution guides can be found [here](https://rdmkit.elixir-europe.org/tool_resource_update).

**Comment by bedroesb** [comment]
I was trying to fix this PR

```
> git push erlefloch pr/erlefloch/1246
To github.com:erlefloch/rdmkit
 ! [remote rejected]   pr/erlefloch/1246 -> pr/erlefloch/1246 (permission denied)
error: failed to push some refs to 'github.com:erlefloch/rdmkit'
```
But I seem to not have the rights

**Comment by bedroesb** [comment]
Let me know if you need any help!

**Comment by erlefloch** [comment]
thanks ! I think the merge conflict is solved now :)

**Comment by bedroesb** [comment]
That was quick!, now the most time consuming is to make use of the new snippet :D

**Comment by bedroesb** [comment]
@erlefloch how far do you think we are from merging in your opinion? :)

**Comment by erlefloch** [comment]
hi @bedroesb , to me it looks ready. However I can see that @floradanna self-assigned the PR a few days ago, so perhaps she has plans to add further modifications ? Anyway, just let me know if there is anything I can improve.

**Review by bedroesb — APPROVED** [review]

**Review by bedroesb — APPROVED** [review]

File: _data/tool_and_resource_list.yml [hunk 3]
```diff
@@ -2279,3 +2279,13 @@
...
   registry:
     biotools: aria_access
+- description: Data Management Plan (DMP) generator that focuses on plant science.
+  id: dataplan
+  name: DataPLAN
+  url: https://plan.nfdi4plants.org
+  registry:
+    biotools: dataplan
+- description: Online questionnaire for the development of data management plans, and DMP repository.
+  id: opidor
```
Comment (bedroesb) [inline _data/tool_and_resource_list.yml:2289]: I just noticed that OPIDoR is already listed under the National resources and is now being duplicated. We will also lose the info that it is an instance of dmproadmap if we take it out of the national resources. Maybe we leave it out @erlefloch ?

**Review by bedroesb — CHANGES_REQUESTED** [review]
Sorry for the confusion, but we try to leave the "instance of" instances out of the main tool list since they are most of the time national flavors. Please do correct me if I am wrong

File: _data/tool_and_resource_list.yml (hunk 3 above)
Comment (erlefloch) [reply _data/tool_and_resource_list.yml:2289]: Sure ! Let's do so

**Comment by erlefloch** [comment]
I applied your suggestion :)

File: _data/tool_and_resource_list.yml (hunk 3 above)
Comment (erlefloch) [reply _data/tool_and_resource_list.yml]: done in https://github.com/erlefloch/rdmkit/commit/6908f58ce731ba8735c44befd5189a640cc47ff6

**Pull request merged**